server/channels/app/platform/busy_test.go

```diff
@@ -119,11 +119,15 @@ func compareBusyState(t *testing.T, busy1 *Busy, busy2 *Busy) bool {
 // ClusterMock simulates the busy state of a cluster.
 type ClusterMock struct {
 	Busy *Busy
+	t    *testing.T
```

Comment:
You also need to assign the `t` when creating a `ClusterMock`

Reply:
Done